mechanicalsoup/form.py

```diff
@@ -277,6 +277,45 @@ def set(self, name, value, force=False):
             return
         raise LinkNotFoundError("No valid element named " + name)
 
+    def __getitem__(self, name):
+        """Forward arguments to :func:`~Form.get` to get value for a given form field.
+        For example, :code:`name = form["name"]` calls :code:`form.get("name")`
+        .
+        """
+        return self.get(name)
+
+    def get(self, name):
+        """Get a ``value`` from input element identified by ``name``.
+        Currently it does support "input" type only.
+
+        Example:
+
+        .. code-block:: python
+            firstname = browser['firstname']
+
+        If the element is not found , raise a :class:`LinkNotFoundError`
+        exception.
+
+"""
+        i = self.form.find("input", {"name": name})
```

Comment:
I think it'd be good to avoid single character variables if possible. Even `inp` instead of `i` would be good here.